app/scripts/components/boilerplate/view-mode.jsx

```diff
@@ -7,6 +7,7 @@ require('stylesheets/boilerplate/view-mode');
 const ViewMode = React.createClass({
   render() {
     return (
+      <div className="view-mode-wrapper">
```

Comment:
lets make this just `view-mode` and rename or remove the `className` from the `<ul>`. Can we reindent the `<ul` and it's contents?